alvr/server_openvr/cpp/alvr_server/bindings.h

```diff
@@ -151,7 +144,7 @@ extern "C" void SetTracking(
     FfiDeviceMotion headMotion,
     FfiHandData leftHandData,
     FfiHandData rightHandData,
-    const FfiBodyTracker* bodyTrackers,
+    const FfiDeviceMotion* bodyTrackers,
```

Comment:
I agree. I will use bodyTrackerMotions. Not bodyMotions, to be clear, because in the future there will be a bodySkeleton (not sure for SteamVR but for Monado).